dae/dae/genomic_resources/tests/test_gene_models.py

```diff
@@ -466,3 +494,60 @@ def test_create_regions_from_genes(
     assert len(result) == 1
     reg = result[0]
     assert str(reg) == expected
+
+
+def test_save_as_gtf_simple(ensembl_gtf_example: GeneModels) -> None:
+    reference = ensembl_gtf_example
```

Comment:
`reference` is not a good name for gene models... it reminds me of a reference genome, not gene models...